sup3r/preprocessing/accessor.py

```diff
@@ -0,0 +1,527 @@
+"""Accessor for xarray."""
+
+import logging
+from typing import Dict, Union
+from warnings import warn
+
+import dask.array as da
+import numpy as np
+import pandas as pd
+import xarray as xr
+from scipy.stats import mode
+from typing_extensions import Self
+
+from sup3r.preprocessing.names import Dimension
+from sup3r.preprocessing.utilities import (
+    _contains_ellipsis,
+    _is_strings,
+    _lowered,
+    _mem_check,
+    _parse_ellipsis,
+    dims_array_tuple,
+    ordered_array,
+    ordered_dims,
+    parse_to_list,
+)
+from sup3r.typing import T_Array
+
+logger = logging.getLogger(__name__)
+
+
+@xr.register_dataset_accessor('sx')
+class Sup3rX:
```

Comment:
can you add a `Sup3rX.values` property that is basically `np.asarray(self[...])`? Not sure if .values would be useful elsewhere but you should consider it... really nice to be able to extract the numpy array from xarray arrays.

Reply:
You can already do `Sup3rX[dset].values` since it acts like an `xr.Dataset` with extras.

Reply:
yeah but `np.asarray(self[...])` does a nice job of stacking the feature channel... can't we just add the .values property? pleaseeeee

Reply:
I added this in the latest update